### ISSUE TITLE
What is episode here in episodic training ?

### ISSUE THREAD

**Issue by shraddha291996** [sicara/easy-few-shot-learning#84]
Hi , 

I have a confusion in understanding how epiosdes are defined in epsiodic training?  I want to know one epoch means how many episodes? also if it is dependent on few shot tasks and if yes, how ? 

Thank you in Advance

**Comment by ebennequin** [comment]
Hi! Did you follow [the tutorial notebook on episodic training](https://github.com/sicara/easy-few-shot-learning/blob/master/notebooks/episodic_training.ipynb)? If so, can you tell me what is unclear?

**Comment by shraddha291996** [comment]
> Hi! Did you follow [the tutorial notebook on episodic training](https://github.com/sicara/easy-few-shot-learning/blob/master/notebooks/episodic_training.ipynb)? If so, can you tell me what is unclear?
Hi thank you for your reply 
yes I followed the notebook , what is unclear to me is  200 epochs means how many episodes? and how are you deciding or calculating that ? 

Thanks in Advance

**Comment by ebennequin** [comment]
The number of episodes is defined in the `train_sampler`. In the notebook we defined the variable `n_tasks_per_epoch = 500`.

An epoch in the context of episodic training is really just the duration between to inferences on the validation set, so the number of tasks per epoch is not really important, what is important is the total number of tasks (or episodes), which is an hyperparameter of the training and should be refined on the validation set or through early stopping (no difference with classical training).
